docs/devGuide/devGuide.md

```diff
@@ -175,12 +175,42 @@ On Windows:
   You should always check that the generated output is correct before committing any changes to the test sites.
 </box>
 
-##### New features
+#### Adding site content
 
 When adding new features, you should also add new site content into an existing test site or create a new test site to demonstrate the new feature. This is to ensure that your feature can be tested by building that test site.
 
+1. Add a new test page, e.g., `newTestPage.md`, containing a demonstration of the new feature.
+
+2. Open `test/functional/test_site/site.json`
+
+3. To include a new page, e.g., `newTestPage.md`, add it to the pages array.
+
+``` {heading="site.json" highlight-lines="16,17"}
+"pages": [
+    {
+      "src": "index.md",
+      "title": "Hello World",
+      "frontmatter": {
+        "frontMatterOverrideProperty": "Overridden by front matter override",
+        "globalAndFrontMatterOverrideProperty":  "Overridden by front matter override"
+      }
+    },
+    ...
+    {
+      "src": "testLayouts.md",
+      "title": "Hello World"
+    },
+    {,
+      "src": "newTestPage.md",
+      "title": [some title you see fit]
+    },
+    ....
+```
+
+4. Update the tests.
+
 <box type="info">
-  When creating a new test site, the directory name of the new test site should be added to <code>test/functional/test_sites</code>.
```

Comment:
This instruction is correct, change it back

Reply:
Agreed. Thanks for clarifying it.